Automattic/wp-calypso
# [E2E][GB] Add test user and site for AT GB stable E2E testing

#### Changes proposed in this Pull Request

Project: p9oQ9f-13V-p2
Follow up to: https://github.com/Automattic/wp-calypso/pull/61783

* Add a new test user and site to the config to be used Gutenberg E2E stable testing on Atomic.

#### Testing instructions

1. Checkout `trunk`
2. Follow the instructions in this fieldguide: PCYsg-vnR-p2 to decrypt the config file
3. Move it to `local-decrypted.trunk.json`
4. Checkout this branch and decrypt the config file again
7. Diff the trunk and the new file: `diff local-decrypted.trunk.json local-decrypted.json` and make sure only the expected changes are there.
8. (Optional) Try running some E2E test to make sure the JSON is valid and is properly loaded

Related to: https://github.com/Automattic/wp-calypso/pull/61432.
